static/js/base.js

```diff
@@ -0,0 +1,32 @@
+$(function() {
+  // http://stepansuvorov.com/blog/2014/04/jquery-put-and-delete/
+  jQuery.each( [ "put", "delete" ], function( i, method ) {
+    jQuery[ method ] = function( url, data, callback, type ) {
+      if ( jQuery.isFunction( data ) ) {
+        type = type || callback;
+        callback = data;
+        data = undefined;
+      }
+   
+      return jQuery.ajax({
+        url: url,
+        type: method,
+        dataType: type,
+        data: data,
+        success: callback
+      });
+    };
+  });
+
+  jQuery.prototype.getDOM = function() {
+    if (this.length === 1) {
+        return this[0];
```

Comment:
Interestingly `[0]` is faster than `.get(0)` https://learn.jquery.com/using-jquery-core/faq/how-do-i-pull-a-native-dom-element-from-a-jquery-object/